delfin/drivers/fake_storage/__init__.py

```diff
@@ -44,15 +45,14 @@
 
 CONF.register_opts(fake_opts, "fake_driver")
 
-
 LOG = log.getLogger(__name__)
 
-
 MIN_WAIT, MAX_WAIT = 0.1, 0.5
 MIN_POOL, MAX_POOL = 1, 100
 MIN_VOLUME, MAX_VOLUME = 1, 2000
 PAGE_LIMIT = 500
-MIN_STORAGE, MAX_STORAGE = 10, 40
+MIN_STORAGE, MAX_STORAGE = 3, 4
+MIN_VALUES, MAX_VALUES = 1, 4
```

Comment:
What is this for? Can these be renamed better like MIN_PERF_VALUES or something more explanatory
Also, what is MIN_STORAGE and MAX_STORAGE and why these values changed?